.github/workflows/periodic-merge.yml

```diff
@@ -32,6 +35,8 @@ jobs:
           permission-pull-requests: write
 
       - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          persist-credentials: false
```

Comment:
Similar to the backport workflow, we pass a token to devmasx/merge-branch in this workflow. Hopefully, that action uses it correctly, without relying on impure git config.

Reply:
The periodic merges still work fine.